ci/courtesy-push.yml

```diff
@@ -59,7 +59,13 @@ steps:
   condition: and(succeeded(), or(eq(variables['WEEK'], '3'), eq(variables['FORCE_COURTESY_PUSH'], 'true')))
 
 - powershell: |
-    $body = '{"text": "Created tasks courtesy push PR. Someone please approve/merge this :please-puss-in-boots: ' + $env:PrLink + '"}'
-    Invoke-RestMethod -Uri $(slackUri) -Method Post -Body $body -ContentType 'application/json'
-  displayName: 'Send Slack notification'
+    $titleText = "Courtesy Bump of Tasks PR created - ID $($env:PrID)"
+    $messageText = "Created Courtesy Bump of Tasks PR. Please review and approve/merge [PR $($env:PrID)]($($env:PrLink)). Related article in [Wiki](https://mseng.visualstudio.com/AzureDevOps/_wiki/wikis/AzureDevOps.wiki/25317/Release-of-pipeline-tasks)."
+    $body = [PSCustomObject]@{
+        title = $titleText
+        text = $messageText
+        themeColor = "#FFFF00"
+      } | ConvertTo-Json
+    Invoke-RestMethod -Uri $(MSTeamsUri) -Method Post -Body $body -ContentType 'application/json'
```

Comment:
As I can see the code for both notifications are quite similar, in this case, could it make sense to extract the coed to the file and reuse it?

Reply:
Done. Code for notifications is moved to separate script.